packages/katello/rubygem-pulp_python_client/rubygem-pulp_python_client.spec

```diff
@@ -2,7 +2,7 @@
 %global gem_name pulp_python_client
 
 Name: rubygem-%{gem_name}
-Version: 3.6.1
+Version: 3.7.2
```

Comment:
https://rubygems.org/gems/pulp_python_client/versions/3.7.3 was released with a correct Faraday requirement.

Reply:
Updated to 3.7.3..The cherry-pick commits were from me trying to clean up some mess in my local branch. Pushed clean commits.

Note: I was unable to add the ostree binding gem. That seems to be special. 🙂

Reply:
I think it's because there is no final release for ostree, just prereleases.